### ISSUE TITLE
not an issue

### ISSUE THREAD

**Issue by jerch** [diamondburned/libdb#1]
A very nice all-in-browser showcase. :smile_cat:

(I know how hard it is to get even most simple convenient shell input semantics right ...)

Just a minor headsup from my side - since version 0.4.0 the image addon supports iTerm's inline image protocol, so the heavy lifting via SIXEL with inferior output quality is not needed anymore. (On how to construct such a sequence, plz see https://iterm2.com/documentation-images.html).

**Comment by diamondburned** [comment]
I prefer SIXEL just because it's the more common protocol out there compared to the mostly-iTerm2-exclusive inline image protocol, even if it has superior image quality.

**Comment by jerch** [comment]
Yeah sixel has the better adoption rate for sure, and we (as TE devs) constantly fail to come up with a better protocol :crying_cat_face:
